installation/standard.py

```diff
@@ -48,8 +48,8 @@
 #     If you set x11libdir (that is two ones and an el) AND x11include to point
 #     to the lib and include directories, they will be used.
 #     Otherwise a search is made for common locations.
-x11libdir=''
-x11include=''
+x11libdir='/usr/X11R6/lib'
+x11include='/usr/X11R6/include'
```

Comment:
> I don't think there is a system libpng besides the xquartz one at least. If we detect that and compile our own libpng and the user later installs xquartz, there is a good chance it will segfault as it did before we started using the xquartz version a while back.

@jbeezley if we compile own own, we will have it our paths and ld paths so that it will be found. The problem may happen if a user try to use another package that uses some other PNG library but that would be hard to avoid. The right way to do this is this:

1. Find system png
2. If found use 1) 
3. If not, build our own.